### PR TITLE
RDKBACCL-293 : Telemetry 2.0 Integration in BPI reference platforms

### PR DESCRIPTION
**Pull request by kpandu987**
Reason for change: Added Telemetry 2.0 support
Test Procedure: Test results are attached in JIRA
Risks: Low

### CODE REVIEW

File: meta-rdk-mtk-bpir4/recipes-ccsp/ccsp/parodus2ccsp/file_perm.patch [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ {
+    appcaps.caps = NULL;
+@@ -17,6 +20,21 @@ void drop_root_privilege()
+    else
+    {
+        WalInfo("NonRoot feature is enabled, dropping root privileges for webpa process\n");
++       FILE *fp = fopen(WEBPA_CFG_FILE,"r");
++       if (fp == NULL)
++       {
++          WalError("WEBPA_CFG_FILE is not present, creating an empty file \n");
```
Comment (cpokuru) [inline meta-rdk-mtk-bpir4/recipes-ccsp/ccsp/parodus2ccsp/file_perm.patch]: is there any reason why we need a patch.can we upstream to parodus2ccsp

File: meta-rdk-mtk-bpir4/recipes-support/parodus/parodus_1.0.bbappend [hunk 2]
```diff
@@ -13,6 +13,7 @@ do_install_append () {
     install -d ${D}${base_libdir_native}/rdk
     install -m 0644 ${S}/devices/broadband/parodus/systemd/parodus.service ${D}${systemd_unitdir}/system
     install -m 0755 ${S}/devices/broadband/parodus/scripts/parodus_start.sh ${D}${base_libdir_native}/rdk
+    sed -i "s/eth0/lan0/g" ${D}${base_libdir_native}/rdk/parodus_start.sh
```
Comment (cpokuru) [inline meta-rdk-mtk-bpir4/recipes-support/parodus/parodus_1.0.bbappend]: can you push this change as part of webpa ticket

File: meta-rdk-mtk-bpir4/recipes-ccsp/ccsp/ccsp-common-library.bbappend [hunk 3]
```diff
@@ -20,16 +20,29 @@ do_install_append_class-target() {
    fi
    install -D -m 0644 ${S}/systemd_units/wan-initialized.target ${D}${systemd_unitdir}/system/wan-initialized.target
    install -D -m 0644 ${S}/systemd_units/wan-initialized.path ${D}${systemd_unitdir}/system/wan-initialized.path
+    #Telemetry support
```
Comment (cpokuru) [inline meta-rdk-mtk-bpir4/recipes-ccsp/ccsp/ccsp-common-library.bbappend:23]: are you able to see results in cloud .https://jira.rdkcentral.com/jira/projects/RDKCLOUD/issues/RDKCLOUD-30?filter=allopenissues

File: meta-rdk-mtk-bpir4/recipes-ccsp/ccsp/parodus2ccsp/file_perm.patch (hunk 1 above)
Comment (kpandu987) [reply meta-rdk-mtk-bpir4/recipes-ccsp/ccsp/parodus2ccsp/file_perm.patch]: In RPI also, we are using this patch .

File: meta-rdk-mtk-bpir4/recipes-support/parodus/parodus_1.0.bbappend (hunk 2 above)
Comment (kpandu987) [reply meta-rdk-mtk-bpir4/recipes-support/parodus/parodus_1.0.bbappend]: ok chandra

File: meta-rdk-mtk-bpir4/recipes-ccsp/ccsp/ccsp-common-library.bbappend (hunk 3 above)
Comment (kpandu987) [reply meta-rdk-mtk-bpir4/recipes-ccsp/ccsp/ccsp-common-library.bbappend:23]: We confirmed ,this functionality is working based on logs .
 As we know, log upload portal is not working for few weeks.
Also, required logs are already uploaded in JIRA .

File: meta-rdk-mtk-bpir4/recipes-ccsp/ccsp/parodus2ccsp/file_perm.patch (hunk 1 above)
Comment (cpokuru) [reply meta-rdk-mtk-bpir4/recipes-ccsp/ccsp/parodus2ccsp/file_perm.patch]: Let me check and come back

**Review by cpokuru — APPROVED** [review]

**Pull request merged**